newsletter-june-2021.rst

```diff
@@ -33,49 +34,52 @@ Company highlights
   (we will write a dedicated blog post about this soon).
 - We added a `Data Processing Agreement <https://docs.readthedocs.io/en/stable/legal/dpa/>`_
   to our official documentation.
-- We are making progress towards a new streamlined workflow
-  that helps us while the company grows.
-
-.. Pageviews stats obtained from Google Analytics, https://readthedocs.io property,
-   and divided by the total number of days in the month
+- We are iterating on an improved roadmap workflow to make sure our team
+  stays productive as we grow.
 
 New features
 ------------
 
 - We added a new configuration key, ``build.apt_packages``,
   that allow our users to install custom operating system packages.
   You can read more `in our dedicated blog post </apt-packages>`_.
-- Now redirects also work `even if you are using
+- Now redirects also work `if you are using
   HTTP <https://github.com/readthedocs/readthedocs.org/issues/8183>`_.
 
 You can always see the latest changes to our platforms in our `Read the Docs
-Changelog <https://docs.readthedocs.io/page/changelog.html>`_ and `Ethical Ad Server
-Changelog <https://ethical-ad-server.readthedocs.io/page/developer/changelog.html>`_.
+Changelog <https://docs.readthedocs.io/page/changelog.html>`_.
 
 Current focus & known issues
```

Comment:
I feel like `Current focus` here is a bit wrong. Maybe we should just make this a "Known issues" section, and put it at the bottom? Features feel more important, but I think it's good to be transparent about struggles as well when we have them.

Reply:
👍🏽